code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -770,19 +790,636 @@ nobiliumsuppression = INFINITY
+		"MAX_TEMP" = 300
+	)
+
+/datum/gas_reaction/proto_nitrate_plasma_response/react(datum/gas_mixture/air, datum/holder)
+	var/energy_released = 0
+	var/old_heat_capacity = air.heat_capacity()
+	var/list/cached_gases = air.gases
+	var/temperature = air.temperature
+	var/turf/open/location = isturf(holder) ? holder : null
+	if(cached_gases[/datum/gas/plasma][MOLES] < 10)
+		var produced_amount = min(5, cached_gases[/datum/gas/plasma][MOLES], cached_gases[/datum/gas/proto_nitrate][MOLES])
+		if(cached_gases[/datum/gas/plasma][MOLES] - produced_amount < 0 || cached_gases[/datum/gas/proto_nitrate][MOLES] - produced_amount * 0.1 < 0)
+			return NO_REACTION
+		cached_gases[/datum/gas/plasma][MOLES] -= produced_amount
+		cached_gases[/datum/gas/proto_nitrate][MOLES] -= produced_amount * 0.1
+		energy_released -= produced_amount * 1500
+		if(prob(produced_amount * 15))
+			new/obj/item/stack/sheet/mineral/plasma(location)
+		if(energy_released)
+			var/new_heat_capacity = air.heat_capacity()
+			if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
+				air.temperature = (temperature * old_heat_capacity + energy_released) / new_heat_capacity
+		return REACTING
+	return NO_REACTION
+
+/datum/gas_reaction/proto_nitrate_bz_response
+	priority = 21
+	name = "Proto Nitrate bz response"
+	id = "proto_nitrate_bz_response"
+
+/datum/gas_reaction/proto_nitrate_bz_response/init_reqs()
+	min_requirements = list(
+		/datum/gas/proto_nitrate = MINIMUM_MOLE_COUNT,
+		/datum/gas/bz = MINIMUM_MOLE_COUNT,
+		"TEMP" = 260,
+		"MAX_TEMP" = 280
+	)
+
+/datum/gas_reaction/proto_nitrate_bz_response/react(datum/gas_mixture/air, datum/holder)
+	var/energy_released = 0
+	var/old_heat_capacity = air.heat_capacity()
+	var/list/cached_gases = air.gases
+	var/temperature = air.temperature
+	var/turf/open/location = isturf(holder) ? holder : null
+	var consumed_amount = min(5, cached_gases[/datum/gas/bz][MOLES], cached_gases[/datum/gas/proto_nitrate][MOLES])
+	if(cached_gases[/datum/gas/bz][MOLES] - consumed_amount < 0)
+		return NO_REACTION
+	if(cached_gases[/datum/gas/bz][MOLES] < 30)
+		radiation_pulse(location, consumed_amount, 1, 1, 0)
+		cached_gases[/datum/gas/bz][MOLES] -= consumed_amount
+	else
+		for(var/mob/living/carbon/L in location)
+			L.hallucination += cached_gases[/datum/gas/bz][MOLES] * 0.5
+	energy_released += 100
+	if(energy_released)
+		var/new_heat_capacity = air.heat_capacity()
+		if(new_heat_capacity > MINIMUM_HEAT_CAPACITY)
+			air.temperature = (temperature * old_heat_capacity + energy_released) / new_heat_capacity
+	return REACTING
+
+/datum/gas_reaction/proto_nitrate_freon_fire_response
+	priority = 22
+	name = "Proto Nitrate freon fire response"
+	id = "proto_nitrate_freon_fire_response"
+
+/datum/gas_reaction/proto_nitrate_freon_fire_response/init_reqs()
+	min_requirements = list(
+		/datum/gas/proto_nitrate = MINIMUM_MOLE_COUNT,
+		/datum/gas/freon = MINIMUM_MOLE_COUNT,
+		/datum/gas/oxygen = MINIMUM_MOLE_COUNT,
+		"TEMP" = 260,
+		"MAX_TEMP" = 300
```

Comment:
Aren't freon fires cold?

Reply:
well yes, but this is not freon fire per sé, this is a reaction between proto nitrate, freon and o2 that will kickstart the fire without having to cool the 2 gases that's why its between 260 and 300 (but i think i'll do it like 270 and 310)